shared/profile/edit-avatar/index.desktop.tsx

```diff
@@ -32,7 +32,7 @@ type State = {
 
 const AVATAR_CONTAINER_SIZE = 175
 const AVATAR_BORDER_SIZE = 4
-const AVATAR_SIZE = AVATAR_CONTAINER_SIZE - AVATAR_BORDER_SIZE * 2
+export const AVATAR_SIZE = AVATAR_CONTAINER_SIZE - AVATAR_BORDER_SIZE * 2
```

Comment:
Is there a better way to do this? For reference, it's being used here https://github.com/keybase/client/pull/23114/files#diff-52fcccf81ad198b87c597588244e00baR5